lib/prompts/expand.js

```diff
@@ -115,7 +115,10 @@ Prompt.prototype.getCurrentValue = function (input) {
     return null;
   }
 
-  return selected.value || selected.name;
+  if (selected.value === undefined) {
```

Comment:
Re-reading the issue, this section here actually is the only bug.

It'll ignore falsy value. As so, that's actually the only fix we need.